src/components/Modals/ReleaseDetailsModal/ReleaseDetailsModalUpgradeOptionsVersion.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import ReleaseDetailsModalUpgradeOptionsBetaLabel from './ReleaseDetailsModalUpgradeOptionsBetaLabel';
```

Comment:
Do we need to repeat the folder path in these components? Could they be just "BetaLabel.tsx" and "Version.tsx"?

Reply:
I know that it's a bit long, but the problem with having a name similar to `Version` for the component and the file is that those names will be used by multiple components. And when you try to use one of these components, 90% of the time TypeScript will automatically import the wrong one 😅